uninstall.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Plugin uninstall file.
+ *
+ * @package AMP
+ */
+
+namespace AmpProject\AmpWP;
+
+// if uninstall.php is not called by WordPress, Then die.
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	die;
+}
+
+/**
+ * Delete data from option table.
+ *
+ * @return void
+ */
+function delete_options() {
+
+	delete_option( 'amp-options' );
+}
+
+/**
+ * Delete AMP Validated URL posts.
+ *
+ * @return void
+ */
+function delete_posts() {
+
+	global $wpdb;
+
+	$current_page = 1;
+	$per_page     = 1000;
+
+	do {
+		$offset = $per_page * ( $current_page - 1 );
+
+		/**
+		 * We don't need to cache result.
+		 * Since we are going to delete those records.
+		 */
+		$result = $wpdb->get_results( // phpcs:ignore
```

Comment:
I'd recommend just ignoring the specific error(s) being reported here so it's more clear what exactly is reported.

Reply:
I have update the comment. Please review.